src/table.js

```diff
@@ -618,8 +618,8 @@ export function __table(source, operations) {
   }
   // Combine column types from schema with user-selected types in operations
   const types = new Map(schema.map(({name, type}) => [name, type]));
-  if (operations.type) {
-    for (const {name, type} of operations.type) {
+  if (operations.types) {
+    for (const {name, type} of operations.types) {
```

Comment:
tiiiny nit: we should update the reference to `operations.type` in the comment on line 632